scripts/stats/cam/exevs_href_precip_stats.sh

```diff
@@ -70,33 +71,33 @@ fi
 #***************************************
 # Build a POE script to collect sub-jobs
 # **************************************
-> run_all_precip_poe.sh
+>$DATA/scripts/run_all_precip_poe.sh
 
 # Build sub-jobs for precip
 if [ $verif_precip = yes ] ; then
  $USHevs/cam/evs_href_precip.sh
  export err=$?; err_chk
- cat ${DATA}/run_all_href_precip_poe.sh >> run_all_precip_poe.sh
+ cat ${DATA}/scripts/run_all_href_precip_poe.sh >> $DATA/scripts/run_all_precip_poe.sh
 fi
 
 # Build sub-jobs for snowfall
```

Comment:
Is there no `scripts/stats/cam/exevs_href_snowfall_stats.sh`? It shared an ex-script with precip?